docs/website/docs/building-from-source/getting-started.md

```diff
@@ -109,6 +109,13 @@ cmake --build ../iree-build/
 
 ### Running tests
 
+Build test dependencies:
+
+``` shell
+cmake --build ../iree-build --target iree-test-deps
```

Comment:
FYI we've thought about renaming this target, since `iree-test-deps` builds deps for specific tests that invoke the compiler at built time (just `tests/e2e/...` IIRC), but not all tests need it. If you skip building that target, you should still be able to run compiler lit tests and runtime cc_tests.

There was a brief discussion about this [on Discord here](https://discord.com/channels/689900678990135345/689906000043573354/949384299814195261), where `iree-test-e2e` was one suggested name to use instead.

(Could be worth revisiting if we're adding more documentation pointing it out)

Reply:
Ack, thanks for the context